modules/javafx.web/src/main/native/Source/WebCore/Modules/compression/CompressionStreamEncoder.cpp

```diff
@@ -77,7 +77,7 @@ ExceptionOr<RefPtr<Uint8Array>> CompressionStreamEncoder::flush()
 bool CompressionStreamEncoder::didDeflateFinish(int result) const
 {
 #if !PLATFORM(JAVA)
-    return !m_zstream.avail_in && (!m_didFinish || (m_didFinish && result == Z_STREAM_END));
+    return !m_zstream.getPlatformStream().avail_in && (!m_didFinish || (m_didFinish && result == Z_STREAM_END));
 #endif
```

Comment:
I presume this came from upstream code? Was it just missed in the earlier set of updates?

In any case, this is a no-op, since it is "ifdef"ed out for our build of WebKit.

Reply:
Yes, this has been missed from the upstream code during update.